### ISSUE TITLE
Blur view weird behavior on Android devices

### ISSUE THREAD

**Issue by MatheusPires99** [Kureev/react-native-blur#404]
I am implementing @react-native-community/blur on version ^3.6.0, and I am having some weird bahavior only on Android devices. 

On iOS it works perfectly fine, but on Android it doesn`t:

![Captura de Tela 2020-10-21 às 22 23 00](https://user-images.githubusercontent.com/49207932/96807247-791bb280-13ec-11eb-86a8-9fbeae613d01.png)
<img width="300" alt="Captura de Tela 2020-10-21 às 22 23 08" src="https://user-images.githubusercontent.com/49207932/96807273-82a51a80-13ec-11eb-958a-de6d47e62b6a.png">

This is my implementation using styled-components, but i have tried to not use it and I am having same the bug

```js
<Content blurType="light" blurRadius={8} overlayColor="transparent">
      ...
</Content>
```

```js
export const Content = styled(BlurView)`
  width: 93%;
  padding: 18px;
  border-radius: 8px;

  position: absolute;
  bottom: 28px;
  left: 12px;
`;
```

**Comment by FelipeYounner** [comment]
+1

**Comment by yalcinozer** [comment]
+1. Same here. Inner components behaves like "absolutely" positioned

**Comment by maksymbevza** [comment]
I just fixed this issue by moving <View> children from BlurView out to be its siblings.
[The example](https://github.com/Kureev/react-native-blur/blob/6897dbcaa5029d9a2c813409b03feb206485a946/example/App.js#L57-L60) says the following.

            BlurView is supported on both iOS and Android.
            If you also need to support Android, the BlurView must be
            absolutely positioned behind your unblurred views, and it
            cannot contain any child views.

There are some mastery needed to make sure they are of the same height. In my case I set `top: 0, bottom: 0` and made sure parent component has the same height as the View that was previously a child of BlurView itself. Not a clean stuff, but works.
